test/e2e/scalability/density.go

```diff
@@ -231,7 +237,10 @@ func runDensityTest(dtc DensityTestConfig) time.Duration {
 	close(logStopCh)
 	framework.Logf("E2E startup time for %d pods: %v", dtc.PodCount, startupTime)
 	framework.Logf("Throughput (pods/s) during cluster saturation phase: %v", float32(dtc.PodCount)/float32(startupTime/time.Second))
+	replicationCtrlStartupPhase.End()
 
+	printPodAllocationPhase := testPhaseDurations.StartPhase(400, "printing pod allocation")
+	defer printPodAllocationPhase.End()
```

Comment:
Please also add a `printPodAllocationPhase.End()` at the end of the code block like others. We don't want this to break in future if we add more code below this.

Reply:
Good point, done.